src/main/java/net/imagej/ops/features/hog/HistogramOfOrientedGradients2D.java

```diff
@@ -69,8 +69,8 @@
  * pixel a histogram of gradient directions by summing up the magnitudes of the
  * neighbored (@param spanOfNeighborhood) pixels. The directions are divided in
  * (@param numOrientations) bins. The output is 3d: for each bin an own channel.
- * Input can be either a grayscale image or a color image with three channels
- * (e.g. RGB, LAB, ...).
+ * Input can be either a grayscale image or a color image with several color
```

Comment:
maybe more like "Input can be either a 2d or 3d image, where the third dimension is interpreted as color (see paper...)"?